include/swift/Driver/ToolChain.h

```diff
@@ -121,6 +121,7 @@ class ToolChain {
     llvm::opt::ArgStringList Arguments;
     std::vector<std::pair<const char *, const char *>> ExtraEnvironment;
     std::vector<FilelistInfo> FilelistInfos;
+    bool allowsResponseFiles = false;
```

Comment:
I think this variable needs a comment explaining why it would ever be true or false.
Something like: "Some platforms limit argument length and those use response files..."

Reply:
It's not just a per-platform decision, so it's not that, but I agree that a comment would be nice.